System/Desk/Desk.ts

```diff
@@ -207,29 +204,22 @@ export class Desk {
         return false;
     }
 
-    setUpContextMenu(body: Element, delegate: any) {
-        return new DeskEvent(body, "contextmenu", (evt: Event) => {
+    setUpContextMenu(body: HTMLElement, delegate: any) {
+        return new DeskEvent(body, "contextmenu", (evt: MouseEvent) => {
             evt.preventDefault();
-            // TODO: spelling
-            // @ts-ignore
             // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-            const list = delegate.prepareContexMenu(body, evt.clientX, evt.clientY);
+            const list = delegate.prepareContextMenu(body, evt.clientX, evt.clientY);
             if (list) {
-                // @ts-ignore
                 // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
                 this.showContextMenu(list, delegate, evt.clientX, evt.clientY);
             }
         });
     }
 
+    // TODO: What is the clipboard type? - Bono
     startDrag(clipboard: any, view: DIView, x: number, y: number, originalX: number, originalY: number) {
-        let i = 0;
-        for (; i < secretaryInstance.mainWorkSpace.apps.length; i++) {
-            if (secretaryInstance.mainWorkSpace.apps[i].allowDrag) {
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-                secretaryInstance.mainWorkSpace.apps[i].dragStart(clipboard);
-            }
-        }
+        secretaryInstance.mainWorkSpace.apps.forEach((app) => app.allowDrag && app.dragStart(clipboard));
```

Comment:
I prefer `for/const` loops over `.forEach` loops

```ts
for (const app of secretaryInstance.mainWorkSpace.apps) {
  if (app.dragStart) app.dragStart(clipboard);
}
```

What are your thoughts @qlqhwb?

Reply:
I do not have preferences, but we can talk about convention in the meeting